src/services/helper/sessions.js

```diff
@@ -426,6 +430,19 @@ module.exports = class SessionsHelper {
 						}
 					})
 				)
+				const userIds = sessionDetails[0].data
+					.map((item) => item.userId.toString())
+					.filter((value, index, self) => self.indexOf(value) === index)
+
+				let mentorDetails = await UserProfileHelper.getAllMentorsDetail(userIds)
+				mentorDetails = mentorDetails.result
+
+				for (let i = 0; i < sessionDetails[0].data.length; i++) {
+					let mentorIndex = mentorDetails.findIndex(
```

Comment:
@ankitpws improve this logic, if store as object, then you can easily access.
sample : {
   "1214123" :{ ...data }
}